README.md

```diff
@@ -56,6 +56,60 @@ It is used by `KnpSnappyBundle` and can be configured according to [their documm
     mkdir -p templates/bundles/SyliusAdminBundle/
     cp -R vendor/sylius/refund-plugin/src/Resources/views/SyliusAdminBundle/* templates/bundles/SyliusAdminBundle/
     ```
+   
+4. If you use sylius v1.8 you also need to change files `src/Entity/Shipping/Shipment.php` and `src/Entity/Order/Adjustment.php` to use proper traits and interfaces
+
+   ```php
+   <?php
+
+   declare(strict_types=1);
+   
+   namespace App\Entity\Order;
+   
+   use Doctrine\ORM\Mapping as ORM;
+   use Sylius\Component\Order\Model\Adjustment as BaseAdjustment;
+   use Sylius\RefundPlugin\Entity\AdjustmentInterface as RefundAdjustmentInterface;
+   use Sylius\RefundPlugin\Entity\AdjustmentTrait;
+   
+   /**
+   * @ORM\Entity
+   * @ORM\Table(name="sylius_adjustment")
+   */
+   class Adjustment extends BaseAdjustment implements RefundAdjustmentInterface
+   {
+       use AdjustmentTrait;
+   }
+   ```
+   
+   ```php 
+   <?php
+
+   declare(strict_types=1);
+   
+   namespace App\Entity\Shipping;
+   
+   use Doctrine\Common\Collections\ArrayCollection;
+   use Doctrine\ORM\Mapping as ORM;
+   use Sylius\Component\Core\Model\Shipment as BaseShipment;
+   use Sylius\RefundPlugin\Entity\ShipmentTrait;
+   use Sylius\RefundPlugin\Entity\ShipmentInterface as RefundShipmentInterface;
+   
+   /**
+   * @ORM\Entity
+   * @ORM\Table(name="sylius_shipment")
+   */
+   class Shipment extends BaseShipment implements RefundShipmentInterface
+   {
+   use ShipmentTrait;
+
+      public function __construct()
+      {
+         parent::__construct();
+         /** @var ArrayCollection<array-key, BaseAdjustmentInterface> $this->adjustments */
```

Comment:
`BaseAdjustmentInterface` is not declared anywhere if I see correctly

Reply:
Well but it works, also same thing happens in Plus 🧙